fe/fe-core/src/main/java/org/apache/doris/analysis/NullLiteral.java

```diff
@@ -101,6 +101,11 @@ public String getStringValue() {
         return "NULL";
     }
 
+    @Override
+    public String getStringValueInFe() {
+        return null;
```

Comment:
so all NullLiteral will return lower case null?

Reply:
btw, u should also add ut to ensure `getStringValueInFe` return right string

Reply:
no, origin logic just data.add(null) , here just keep origin logic